Helper/SeedData.cs

```diff
@@ -0,0 +1,37 @@
+﻿using Helpers.Http;
```

Comment:
Is SeedData.cs being used by any other projects? If not move it to PcsConfig

Reply:
@jillcary Do you need it? I suppose yes.